llvm/lib/Target/PowerPC/PPCFrameLowering.cpp

```diff
@@ -2740,3 +2740,11 @@ bool PPCFrameLowering::enableShrinkWrapping(const MachineFunction &MF) const {
     return false;
   return !MF.getSubtarget<PPCSubtarget>().is32BitELFABI();
 }
+
+uint64_t PPCFrameLowering::getStackThreshold() const {
+  // The scratch register of STUX contains a signed negative 64-bit number.
```

Comment:
I'm not very familiar with ppc, but this comment doesn't mean much to me. Maybe other reviewers with more familiarity can triple check it?